<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutGen">
       <item row="0" column="0">
        <widget class="QLabel" name="labelUpdateInterval">
         <property name="text">
          <string>Update interval:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBoxUpdateInterval">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="specialValueText">
          <string>disabled</string>
         </property>
         <property name="suffix">
          <string> min</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>1440</number>
         </property>
         <property name="singleStep">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupAppearance">
         <property name="title">
          <string>Appearance</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxTemp">
            <property name="text">
             <string>Temperature</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxWindChill">
            <property name="text">
             <string>Wind chill</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxWind">
            <property name="text">
             <string>Wind</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBoxWindGust">
            <property name="text">
             <string>Wind gust</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBoxPress">
            <property name="text">
             <string>Pressure</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkBoxVisib">
            <property name="text">
             <string>Visibility</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupUnits">
         <property name="title">
          <string>Units</string>
         </property>
         <layout class="QFormLayout" name="formLayoutUnits">
          <item row="0" column="0">
           <widget class="QLabel" name="labelTemp">
            <property name="text">
             <string>Temperature:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="groupTemp" native="true">
            <layout class="QHBoxLayout" name="horizontalLayoutTemp">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButtonTempC">
               <property name="text">
                <string>°C</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonTempF">
               <property name="text">
                <string>°F</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonTempK">
               <property name="text">
                <string>K</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelWind">
            <property name="text">
             <string>Wind:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="groupWind" native="true">
            <layout class="QHBoxLayout" name="horizontalLayoutWind">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButtonWindKMH">
               <property name="text">
                <string>km/h</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonWindMPH">
               <property name="text">
                <string>MpH</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonWindMS">
               <property name="text">
                <string>m/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelPress">
            <property name="text">
             <string>Pressure:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QWidget" name="groupPress" native="true">
            <layout class="QHBoxLayout" name="horizontalLayoutPress">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButtonPressHPA">
               <property name="text">
                <string>hPa</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonPressPSI">
               <property name="text">
                <string>psi</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonPressMMHG">
               <property name="text">
                <string>mmHg</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabServices">
      <attribute name="title">
       <string>Services</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutApp">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupGoogle">
         <property name="title">
          <string>Google Search</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupWUnd">
         <property name="title">
          <string>Weather Underground</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayoutWUnd">
          <item row="0" column="0">
           <widget class="QLabel" name="labelWUndKey">
            <property name="text">
             <string>API Key:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditWUndKey"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelWUndLocation">
            <property name="text">
             <string>Location:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditWUndLocation">
            <property name="toolTip">
             <string>&lt;table&gt;
&lt;tr&gt;&lt;td&gt;&lt;tt&gt;blank input&lt;/tt&gt;&lt;/td&gt;&lt;td align=&quot;right&quot;&gt;&lt;nobr&gt;IP based geo localization&lt;/nobr&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;&lt;tt&gt;country/city&lt;/tt&gt;&lt;/td&gt;&lt;td align=&quot;right&quot;&gt;&lt;i&gt;&lt;nobr&gt;(e.g. Australia/Sydney)&lt;/nobr&gt;&lt;/i&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;&lt;tt&gt;latitude,longitude&lt;/tt&gt;&lt;/td&gt;&lt;td align=&quot;right&quot;&gt;&lt;i&gt;&lt;nobr&gt;(e.g. 51.5072,-0.1275)&lt;/nobr&gt;&lt;/i&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;&lt;tt&gt;airport code&lt;/tt&gt;&lt;/td&gt;&lt;td align=&quot;right&quot;&gt;&lt;i&gt;&lt;nobr&gt;(e.g. KJFK)&lt;/nobr&gt;&lt;/i&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;&lt;tt&gt;PWS id&lt;/tt&gt;&lt;/td&gt;&lt;td align=&quot;right&quot;&gt;&lt;i&gt;&lt;nobr&gt;(e.g. pws:KCASANFR70)&lt;/nobr&gt;&lt;/i&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;/table&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupYahoo">
         <property name="title">
          <string>Yahoo Weather</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayoutYahoo">
          <item row="0" column="0">
           <widget class="QLabel" name="labelYahooLocation">
            <property name="text">
             <string>Location:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditYahooLocation">
            <property name="toolTip">
             <string>&lt;table&gt;
&lt;tr&gt;&lt;td&gt;&lt;tt&gt;location&lt;/tt&gt;&lt;/td&gt;&lt;td align=&quot;right&quot;&gt;&lt;i&gt;&lt;nobr&gt;(e.g. Sydney, AU)&lt;/nobr&gt;&lt;/i&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;&lt;tt&gt;WOEID&lt;/tt&gt;&lt;/td&gt;&lt;td align=&quot;right&quot;&gt;&lt;i&gt;&lt;nobr&gt;(e.g. 44418)&lt;/nobr&gt;&lt;/i&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;/table&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMisc">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutMisc">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupCommand">
         <property name="title">
          <string>Custom action command</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutCmd">
          <item>
           <widget class="QLineEdit" name="lineEditCommand">
            <property name="toolTip">
             <string>&lt;p&gt;Supported placeholders:&lt;/p&gt;
&lt;table&gt;
&lt;tr&gt;&lt;td&gt;&lt;tt&gt;%L&lt;/tt&gt; - &lt;/td&gt;&lt;td align=&quot;right&quot;&gt;&lt;nobr&gt;user defined location&lt;/nobr&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;&lt;tt&gt;%%&lt;/tt&gt; - &lt;/td&gt;&lt;td align=&quot;right&quot;&gt;&lt;nobr&gt;% character&lt;/nobr&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;/table&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>spinBoxUpdateInterval</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
